python/seldon_core/batch_processor.py

```diff
@@ -287,6 +287,68 @@ def _start_request_worker(
         q_in.task_done()
 
 
+def _multi_request_extract_raw_data(loaded_data, tags):
```

Comment:
Probably should make sure is typed (for consistency)

Reply:
done